SECURITY.md

```diff
@@ -0,0 +1,10 @@
+# Security Policy
+
+## Supported Versions
+
+Version | Supported Until
+--- | ---
+<= 1.2 | Unsupported
+1.0 | Unsupported
```

Comment:
Note that there is no 1.2 branch, so these two lines could be replaced by:
```
< 2.0 | Unsupported
```

Reply:
+1 Thanks :-)